libnavigation-metrics/src/test/java/com/mapbox/navigation/metrics/MapboxMetricsReporterTest.kt

```diff
@@ -72,9 +85,13 @@ class MapboxMetricsReporterTest {
     }
 
     private fun initMetricsReporterWithTelemetry(): MapboxTelemetry {
+        val parentJob = SupervisorJob()
+        val testScope = CoroutineScope(parentJob + coroutineRule.testDispatcher)
+        mockkObject(ThreadController)
```

Comment:
better to move `mockkObject(ThreadController)` to `@Before` and  `unmockkObject` to `@After` func